File: Mobile_Coffee/Project-Mobile__Coffe-Store/app/src/main/java/com/nlu/packages/dotnet_callapi/responsedto/ProductRespondeDTO.java
package com.nlu.packages.dotnet_callapi.responsedto;

import java.io.Serializable;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class ProductRespondeDTO implements Serializable {
    private int id;
    private String name;
    private String type;
    private double basePrice;
    private int quantities;
    private boolean active;
    private int categoryId;
    private String avatarUrl;
    private List<IngredientRespondeDTO> ingredients;
}

File: REST_DotNET_Coffee_Android/DTO/responde/ProductResponde/ProductRespondeDTO.cs
#nullable disable

public class ProductRespondeDTO
{
    public int Id { get; set; }

    public string Name { get; set; } = String.Empty;

    public string Type { get; set; } = string.Empty;

    public double BasePrice { get; set; }

    public int Quantities { get; set; }

    public bool Active { get; set; }

    public int CategoryId { get; set; }

    public string AvatarUrl { get; set; } = String.Empty;

    public List<IngredientRespondeDTO> Ingredients { get; set; }
}